sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -161,17 +161,16 @@ internal class AmqpSender : TransportSender
             CreateMessageBatchOptions options,
             CancellationToken cancellationToken)
         {
-            TransportMessageBatch messageBatch = null;
-            Task createBatchTask = _retryPolicy.RunOperation(async (timeout) =>
-            {
-                messageBatch = await CreateMessageBatchInternalAsync(
-                    options,
-                    timeout).ConfigureAwait(false);
-            },
-            _connectionScope,
-            cancellationToken);
-            await createBatchTask.ConfigureAwait(false);
-            return messageBatch;
+            return await _retryPolicy.RunOperation(static async (value, timeout, _) =>
+                {
+                    var (sender, ops) = value;
```

Comment:
nit:  It's not clear to me what `ops` is without looking at the original context.   Can we go with `options` again?

Reply:
Same comment as abobeu